crates/sui-graphql-rpc/src/types/validator_set.rs

```diff
@@ -27,3 +32,40 @@ pub(crate) struct ValidatorSet {
     // validator_candidates: Option<MoveObject>,
     pub validator_candidates_size: Option<u64>,
```

Comment:
Just noticed that there are commented out fields here that need to be implemented.

@emmazzz -- could you mark these with TODOs and put the workstream in the backlog? (I would say it's a P1).

Reply:
Indeed... I didn't even notice these were commented out fields - I thought they were just doc comments for the next field haha. Will add them to the backlog